### PR TITLE
TE5-T637:VueJs-Practical-02

### PR DESCRIPTION
**Pull request by JeetMakadiya**

### CODE REVIEW

**Comment by JeetMakadiya** [comment]
VueJs Practical-02 Completed 90%.
Screenshot from 2022-03-29 11-57-23

File: practical-02/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (WebDev-Auction) [inline practical-02/.gitignore]: add new line

File: practical-02/src/components/NavBar.vue [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<template>
+  <header class="bg-primary pt-2 pb-2">
+    <h1 class="text-center text-light">Car Showroom</h1>
```
Comment (WebDev-Auction) [inline practical-02/src/components/NavBar.vue]: use data property

**Comment by JeetMakadiya** [comment]
All card with same size
Screenshot from 2022-03-30 10-44-07

**Pull request merged**